learn/os_setup/install_gfortran.md

```diff
@@ -95,6 +95,17 @@ Install a gcc version:
 ```bash
 sudo port install gcc10
 ```
+
+## OpenBSD
+```bash
+pkg_add g95
+```
+
+On OpenBSD, the GFortran executable is named `egfortran`. To test it, type:
+```bash
+egfortran -v
```

Comment:
Is this an issue for fpm to discover the right compiler? It probably requires to set `--compiler egfortran` on every as well.

Reply:
@awvwgk, yes, `fpm` works with `--compiler egfortran`.  `export FPM_COMPILER=egfortran` also works 👍.